lib/simulator-xcode-9.js

```diff
@@ -64,6 +67,13 @@ class SimulatorXcode9 extends SimulatorXcode8 {
     if (opts.scaleFactor) {
       opts.devicePreferences.SimulatorWindowLastScale = parseFloat(opts.scaleFactor);
     }
+    if (opts.simulatorWindowCenter) {
```

Comment:
I still would prefer this option to be written directly into _devicePreferences_